docs/cyber/CyberRT_FAQs.md

```diff
@@ -79,5 +79,18 @@ You can refer the sample code:
 - talker: https://github.com/gruminions/apollo/blob/record/cyber/examples/talker.cc
 - listener: https://github.com/gruminions/apollo/blob/record/cyber/examples/listener.cc
 
+## How to configure multiple hosts communication?
+
+Make sure the two(or more) hosts under the same network segment of the local area network, Like `192.168.10.6` and `192.168.10.7`.
```

Comment:
Make sure that the hosts (two or more) are under the same network segment of the local area network, Like `192.168.10.6` and `192.168.10.7`.

Reply:
done